framework/web/JsonResponseFormatter.php

```diff
@@ -38,6 +38,11 @@
 class JsonResponseFormatter extends Component implements ResponseFormatterInterface
 {
     /**
+     * @var string|null the MIME type used in the `Content-Type` header of the response.
+     * When equals `null` default MIME type will be used based on the `useJsonp` property.
+     */
+    public $mimeType = null;
```

Comment:
Someone may try to set it to the application/javascript; charset=UTF-8 which will result in error.